pyart/graph/radardisplay.py

```diff
@@ -122,21 +150,6 @@ def __init__(self, radar, shift=(0.0, 0.0)):
             warnings.warn('RadarDisplay does not correct for moving platforms')
         self.loc = (lat, lon)
```

Comment:
Setting class attributes is typically not recommended outside of the __init__ method, but I think it this case it makes sense.  Sometime in the future I want to make calculations of the x, y, and z coordinates be performed on a per sweep basis just prior to plotting.